misc/languages/data/uni_loadregiongroups.sql

```diff
@@ -0,0 +1,145 @@
+create table if not exists uni_regiongroups (language char, country char, primary key(language, country));
```

Comment:
Just to help me understand: was this created by hand or does this information come from somewhere?

I.e. how should this file be maintained/kept up to date?

Reply:
Yes, the table of region groups was created manually. It affects only 5 languages (Arabic, English, Spanish, Portuguese, and Chinese). It is based on Unicode data, but it is not trivial to extract the relevant data from the Unicode files.

My guess is that these lists are quite static and will not change often. However, I will check whether the lists can be generated automatically with reasonabl effort.